source/agora/node/FullNode.d

```diff
@@ -466,4 +486,66 @@ public class FullNode : API
     protected void onAcceptedTransaction () @safe
     {
     }
+
+    /***************************************************************************
+
+        Push the block data to the `block_handlers` target server list
+        set in config.
+
+        Convert block data to JSON serialization and send it POST using Rest.
+
+        Params:
+            block = externalized block data
+
+    ***************************************************************************/
+
+    public void pushBlock (const ref Block block) @safe
+    {
+        foreach (handler_address; this.block_handlers.byKey)
```

Comment:
I think this would be better.

```suggestion
        foreach (address, handler; this.block_handlers)
```
Link to reference)
https://dlang.org/spec/hash-map.html#aa_example_iteration

Reply:
Good! This looks better.